docs/developer-guide/ksqldb-clients/java-client/api/allclasses-index.html

```diff
@@ -0,0 +1,300 @@
+<!DOCTYPE HTML>
+<!-- NewPage -->
+<html lang="en">
+<head>
+<!-- Generated by javadoc (11.0.8) on Mon Apr 19 21:22:19 CDT 2021 -->
+<title>All Classes (ksqldb-api-client 0.17.0-1 API)</title>
```

Comment:
What was the outcome on the discussion regarding versioning this release as `0.17.0-1` vs `0.17.0`? I think for consistency it's better to use `0.17.0`, in which case we'll need to regenerate the release artifacts, and this version here will also be updated.